### PR TITLE
add setuptools_scm as build dependency

### LINKED ISSUE #20: `plotnine.__version__` reports 0.0.0 as version
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Try the following:
```bash
conda install plotnine
conda list | grep -i plotnine
# plotnine                  0.10.1             pyhd8ed1ab_1    conda-forge
python3 -c "import plotnine; print(plotnine.__version__)"
# 0.0.0
```
As you can see, `0.0.0` != `0.10.1`

### Installed packages

```shell
plotnine                  0.10.1             pyhd8ed1ab_1    conda-forge
```

### Environment info

```shell
active environment : default
    active env location : /opt/anaconda/envs/default
            shell level : 1
          conda version : 22.11.1
    conda-build version : 3.23.3
         python version : 3.9.15.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.37=0
                          __linux=6.1.12=0
                          __unix=0=0
       base environment : /opt/anaconda  (writable)
      conda av data dir : /opt/anaconda/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
          package cache : /opt/anaconda/pkgs
       envs directories : /opt/anaconda/envs
               platform : linux-64
             user-agent : conda/22.11.1 requests/2.28.1 CPython/3.9.15 Linux/6.1.12-arch1-1 arch/ glibc/2.37
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

### PR DESCRIPTION
**Pull request by Hoeze**
Try to fix #20 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Hoeze** [comment]
@conda-forge-admin, please rerender

**Comment by Hoeze** [comment]
Depends on https://github.com/conda-forge/mizani-feedstock/pull/34

**Comment by Hoeze** [comment]
@conda-forge-admin, please restart ci

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,6 +36,11 @@ requirements:
     # - geopandas >=0.3.0
 
 test:
+  requires:
+    - pip
+    - pytest
```
Comment (has2k1) [inline recipe/meta.yaml:41]: I don't think it is good idea to run tests for plotnine. `scikit-misc` which is required for the tests is not on conda-forge yet. I have just moved it meson-build and hopefully that will make it easy to put it on conda-forge soon.

Comment (Hoeze) [reply recipe/meta.yaml:41]: Alright, I disabled pytest for now :)

**Comment by Hoeze** [comment]
@has2k1 this is ready to be merged

**Comment by has2k1** [comment]
@Hoeze, thank you a bunch.

**Pull request merged**